cfme/tests/services/test_service_catalogs.py

```diff
@@ -27,11 +27,11 @@
 def pytest_generate_tests(metafunc):
     # Filter out providers without provisioning data or hosts defined
     argnames, argvalues, idlist = testgen.infra_providers(metafunc,
-        required_fields=[
-            ['provisioning', 'template'],
-            ['provisioning', 'host'],
-            ['provisioning', 'datastore']
-        ])
+                                                          required_fields=[
```

Comment:
my personal opinion is, that just putting metafunc together with required_fields on the next line
may be more readable than aligning to deep indentation,